README.md

```diff
@@ -6,6 +6,21 @@ Code formatting is maintained via
 sbt scalafmt Test/scalafmt
 ```
 
+## Diverse notes
+By default, diffgraph application, deserialization from storage, and serialization to storage are all multi-threaded.
+
+This can be globally disabled via `flatgraph.Misc.force_singlethreaded()`, for easier debugging. 
+
+In order to quickly glance the input of flatgraph files, you can extract the manifest json with `tail`, e.g. `tail someGraph.fg | jless`:
```

Comment:
```suggestion
In order to quickly glance the input of flatgraph files, you can extract the manifest json with `tail`, e.g. `tail someGraph.fg`:
```

I wouldn't include `|jless` because 1) it's not very commonly installed and 2) it fails if tail happens to also output the binary graph data beforehand...

Reply:
> 2) it fails if tail happens to also output the binary graph data beforehand...

It now outputs the necessary newlines to prevent that ;)

Enough even for people who just use `tail` instead of `tail -n 1`. I'm really proud of that tiny quality-of-life improvement.

Reply:
:) ok very convincing